### ISSUE TITLE
Create Pinot Tables

### ISSUE THREAD

**Issue by Alberter** [pinpoint-apm/pinpoint#11332]
When I use the script multi table. sh to create the pinot table, an error of 500 is reported
![image](https://github.com/user-attachments/assets/e753b322-91b5-4559-99fc-dd4284570e3b)

**Comment by Alberter** [comment]
I had start Apache Zookeeper, Pinot Controller, Pinot Broker, and Pinot Server.(https://docs.pinot.apache.org/basics/getting-started/running-pinot-locally)

**Comment by minwoo-jung** [comment]
@donghun-cho please review this issue.

**Comment by donghun-cho** [comment]
The pinot table won't be created if pinot cluster cannot connect to Kafka or cannot find the kafka topic.

can the pinot cluster access kafka?

**Comment by Alberter** [comment]
Thank you for your clarification. Two Kafka services were started locally, and one was shut down to solve the problem.
But now there is another issue, the document did not find the steps on how to create a hybrid table. I first created a real-time table, but when creating the offline table, an error message was reported stating that the table already exists

**Comment by donghun-cho** [comment]
> The pinot table won't be created if pinot cluster cannot connect to Kafka or cannot find the kafka topic.

This applies to realtime table only
Offline tables can be created without requiring a Kafka connection. so it can be already created from multi-table script

A Pinot Minion is also required for hybrid tables. https://docs.pinot.apache.org/operators/operating-pinot/pinot-managed-offline-flows

**Comment by Alberter** [comment]
Using API/brokers/tables to view tables, systemMetricDouble does not exist, but executing point-admin.sh addTable yields an error
![image](https://github.com/user-attachments/assets/95b4f5d4-b6cf-45b0-933c-3cf7f3d7c3b6)
@donghun-cho

**Comment by Alberter** [comment]
![image](https://github.com/user-attachments/assets/062a6b2e-6450-4e3c-ac61-1762cc75559f)

**Comment by donghun-cho** [comment]
https://docs.pinot.apache.org/basics/concepts/components/table#offline-table-creation
Instead of `Using launcher scripts`
try using `curl` or use swagger ui  
![image](https://github.com/user-attachments/assets/6931a23b-9ac1-4a93-b282-2219c34ae174)

**Comment by Alberter** [comment]
The offline table was successfully created, but the creation of the real-time table still failed with an error message as shown in the screenshot above

**Comment by Alberter** [comment]
The offline table was successfully created using curl, but it seems that the real-time table does not support curl creation. Creating it using pinpoint-admin.sh still fails:
![image](https://github.com/user-attachments/assets/5d7e44a8-d741-4171-938c-3d8b8e3cb360)

**Comment by donghun-cho** [comment]
Could you check which table (realtime or offline) has been created?
use pinot controller, or use API/tables with types(not API/brokers/tables)

**Comment by Alberter** [comment]
The offline table has been successfully created.The real-time table has not been created yet.
![image](https://github.com/user-attachments/assets/2f9726cf-ccca-4505-afcd-5e44584e8cc3)
![image](https://github.com/user-attachments/assets/06d117d0-9cff-4ca4-a29c-f4ac43a1bc8f)

**Comment by donghun-cho** [comment]
try using curl to create realtime.
multi-table.sh used to create inspectorStatAgent00_REALTIME use curl.

or delete systemMetricDouble schema and offline table and recreate tables by following steps 
1. create schema and realtime table using pinot-admin.sh
2. add offline table was using curl

**Comment by Alberter** [comment]
When creating a real-time table using curl, the following error occurs
![image](https://github.com/user-attachments/assets/c695cc86-e3d7-4991-9c62-3b3f955b1fcd)
![image](https://github.com/user-attachments/assets/5495006a-c781-4fc1-a7b7-26ae4e48063d)

**Comment by Alberter** [comment]
![image](https://github.com/user-attachments/assets/47cd67bc-8ea0-4c47-8fa4-2fc3e2937d22)

**Comment by Alberter** [comment]
Using Pinpoint's built-in Kafka and switching to Apache Kafka resulted in the issue

**Comment by donghun-cho** [comment]
No, because the application table uses Pinot ingestion aggregations and already has good response times compared to the agent table
